test/plugin/test_out_google_cloud.rb

```diff
@@ -70,6 +70,70 @@ def test_server_error
     assert_equal 1, exception_count
   end
 
+  def test_prometheus_requests_count
+    setup_gce_metadata_stubs
+    [
+      # Single successful request.
+      [200, 1, 1, 0],
+      # Several successful requests.
+      [200, 2, 2, 0],
+      # Single failed request.
+      [401, 1, 0, 1]
+    ].each do |code, emit_count, successful_count, failed_count|
+      setup_prometheus
+      stub_request(:post, WRITE_LOG_ENTRIES_URI)
+        .to_return(status: code, body: 'Some Message')
+      (1..emit_count).each do |i|
+        d = create_driver(PROMETHEUS_ENABLE_CONFIG)
```

Comment:
If I run it only once, I don't know how to make several entries with the same tag to be batches into the separate requests

If I run it every time, but don't recreate the driver, entries emitted earlier are still there and change the metric values